lib/resource/ApiKey.js

```diff
@@ -6,39 +6,38 @@ var InstanceResource = require('./InstanceResource');
 var utils = require('../utils');
 
 /**
- * @class ApiKey
```

Comment:
You should restore the spaces that you removed from this block comment. For valid JSDoc block comment formatting, please see examples at: http://usejsdoc.org/about-getting-started.html